recipes/vcsh/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I think this file can be removed as it is available from the source tarball.

Reply:
OK.